src/lib/y2users/user.rb

```diff
@@ -154,7 +162,7 @@ def uid?
     def ==(other)
       return false unless other.is_a?(self.class)
 
-      [:name, :uid, :gid, :shell, :home, :gecos, :source, :password].all? do |a|
+      [:name, :uid, :gid, :shell, :home, :gecos, :source, :password, :authorized_keys].all? do |a|
```

Comment:
I think that *authorized_keys* array has to be duplicated when coping a user (`#copy` method). Otherwise:

~~~
user1.authorized_keys = ["key1", "key2"]
user2 = user1.copy
user2.authorized_keys << "key3"
user1.authorized_keys #=> ["key1", "key2", "key3"]
~~~

Reply:
Something like:

~~~
def copy(other)
  ...
  authorized_keys = other.authorized_keys.map(&:dup)
  ...
end
~~~

Reply:
Ah, good point. I overlooked it.